### PR TITLE
Cs branch

### PR DESCRIPTION
**Pull request by schizza**
HassTurnOn/off sentences.

### CODE REVIEW

File: sentences/cs/_common.yaml [hunk 1]
```diff
@@ -7,11 +7,60 @@ responses:
...
+  temperature_unit:
+    values:
+      - "(celsius | celsia)"
+      - in: "c"
+        out: "Celsia"
+      - "(fahrenheit | fahrenheita)"
+      - in: "f"
+        out: "Fahrenheita"
 expansion_rules:
+  area: "{area}"
```
Comment (halecivo) [inline sentences/cs/_common.yaml:50]: trivial expansion is not needed

File: sentences/cs/_common.yaml [hunk 2]
```diff
@@ -7,11 +7,60 @@ responses:
...
-  vypnout: "(vypni|vypnout)"
+  vypnout: "(vypni|vypnout|zastavit|zastav)"
   rozsvitit: "(rozsviť|rozsvítit)"
   zhasnout: "(zhasni|zhasnout)"
+  nastav: "(nastav|nastavit)"
+  teplota: "{temperature} stupňů {temperature_unit}"
+  brightness: "{brightness} [procent]"
+  zavri: "(zavři | zavřít)"
+  otevri: "(otevři | otevřít)"
+  garaz: "garáž"
```
Comment (halecivo) [inline sentences/cs/_common.yaml:61]: trivial expansion is not needed, also please let's not add unused expansions. To explain why - Your PR is about HassTurnOn/Off, if somebody starts working on curtain, he will add his expansion rules, which may be different from the ones you entered. It also helps to keep PR small.

**Review by halecivo — CHANGES_REQUESTED** [review]
Please also use more informative PR name - here it could be for example "czech HassTurnOn/Off". This helps others to see what is being worked on, so they can pick something else. Thanks!

PS: There is already a PR for HassTurnOn/Off, but since not even tests are passing there, it's a question whether author will finish it.

File: sentences/cs/_common.yaml (hunk 1 above)
Comment (schizza) [reply sentences/cs/_common.yaml:50]: Ok, but might be needed in future:  area: "[(na|v)] {area}"
```suggestion
  area: "[(na|v)] {area}"
```

File: sentences/cs/_common.yaml (hunk 2 above)
Comment (schizza) [reply sentences/cs/_common.yaml:61]: Point taken, but I have all sentences done and now working on tests. That's why there are here. Will post test Giles soon for other domains.

File: sentences/cs/_common.yaml (hunk 1 above)
Comment (halecivo) [reply sentences/cs/_common.yaml]: Okay, like this it would make sense.

File: sentences/cs/_common.yaml (hunk 2 above)
Comment (halecivo) [reply sentences/cs/_common.yaml:61]: Okay, but it is difficult to review unused expansion rules. For garáž (garage), I would say it is a cover device or an area. 
Of course I cannot know for sure, if I don't see the usage.

Comment (schizza) [reply sentences/cs/_common.yaml]: Ok, cleaned.

File: sentences/cs/_common.yaml [hunk 3]
```diff
@@ -7,11 +7,56 @@ responses:
...
+        out: "Fahrenheita"
 expansion_rules:
+  area: "[(na|v)] {area}"
   zapnout: "(zapni|zapnout)"
-  vypnout: "(vypni|vypnout)"
+  vypnout: "(vypni|vypnout|zastavit|zastav)"
   rozsvitit: "(rozsviť|rozsvítit)"
   zhasnout: "(zhasni|zhasnout)"
+  nastav: "(nastav|nastavit)"
+  teplota: "{temperature} [stupňů] {temperature_unit}"
```
Comment (halecivo) [inline sentences/cs/_common.yaml:56]: Better would be 
`{temperature} [stupňů] [{temperature_unit}]`
Your version allow to say "15 Celsia", but not "15 stupňů", which is quite common.
Not a critical comment, however.

File: sentences/cs/homeassistant_HassTurnOff.yaml [hunk 4]
```diff
@@ -2,4 +2,6 @@ language: cs
 intents:
   HassTurnOff:
     data:
-    - sentences: []
+      - sentences: 
+          - "<vypnout> <name>"
```
Comment (halecivo) [inline sentences/cs/homeassistant_HassTurnOff.yaml:6]: Name is not expansion rule, therefore I think the tests will fail here.
`<vypnout> {name}` should work.

**Review by halecivo — CHANGES_REQUESTED** [review]

File: sentences/cs/_common.yaml (hunk 3 above)
Comment (schizza) [reply sentences/cs/_common.yaml:56]: Yes, found it when testing climate :)
Corrected.

**Comment by schizza** [comment]
Now I don't know if I need to apply for PR?

**Comment by halecivo** [comment]
The problem is that you pushed to the same branch another change, therefore this PR had been updated and is no more focused on a single domain.

If you start a new work, you need to create a new branch, so you can have separate PR.

In Github desktop, you can create new branch from your current state, push it, then switch back to the old branch (cs-branch), drop/revert commits for different domain than HassTurnOn/Off.

**Comment by halecivo** [comment]
Well, since #279 has been merged right away, this PR can be abandoned.

**Pull request closed without merging**